### ISSUE TITLE
Hello, The Theme has a small problem on my terminal

### ISSUE THREAD

**Issue by netcan** [bhilburn/powerlevel9k#71]
![screenshot - 2015 08 08 - 11 46 39](https://cloud.githubusercontent.com/assets/11377070/9148927/27bb3d2a-3dc3-11e5-8683-ce8c8244dca7.png)
the theme is so cool but on my terminal isn't perfect. I install the awesome powerline fonts, and arg `POWERLEVEL9K_MODE='awesome-patched'`

**Comment by dritter** [comment]
Hi @netcan 
What terminal emulator do you use?

You seem to have two problems:

1. your terminal doesn't support 256 colors. Try to set `TERM=xterm-256colors`, or similar.

2. your terminal seems to use the wrong font. Have you chosen an awesome-patched font in the settings of your terminal emulator?

**Comment by netcan** [comment]
I use the terminal is gnome-terminal，and Built-In schemes is XTerm，in ~/.zshrc，I `export TERM="xterm-256color"`，and `POWERLEVEL9K_MODE='awesome-patched'` but dosen't work.

**Comment by dritter** [comment]
@netcan Yep. For this theme you need a special font.

If you use `awesome-patched`, you need a [awesome-font](https://github.com/gabrielelana/awesome-terminal-fonts).
In all other cases, you need an [powerline-font](https://github.com/powerline/fonts)

That should do the trick ;)

**Comment by netcan** [comment]
![screenshot - 08102015 - 10 04 50 am](https://cloud.githubusercontent.com/assets/11377070/9158339/871c9c7a-3f47-11e5-9e53-9ad6209b703f.png)
![Uploading Screenshot - 08102015 - 10:04:50 AM.png…]()
But it look like dosen't work...and this is my ~/.zshrc 
![screenshot - 08102015 - 10 07 47 am](https://cloud.githubusercontent.com/assets/11377070/9158343/ae11e7a4-3f47-11e5-973c-c2d7936e99eb.png)

**Comment by dritter** [comment]
This is still the wrong font.

To clear things up:
In `awesome-patched` mode you need either a font that combines the original awesome font, octoicons and a powerline-patched font _or_ a mechanism to use a normal font and fall back to the right one. The `awesome-terminal-fonts` repository suggests the latter way, but offers in its `patching-strategy` a already combined font (first way).
If the second way doesn't work for you, just download one of the already patched fonts [here](https://github.com/gabrielelana/awesome-terminal-fonts/tree/patching-strategy/patched).

I assume, you just installed the "awesome-font" from the "fonts" directory, which is just the normal awesome-font and does not include octoicons and powerline..

Let me know, if that worked. :)

**Comment by netcan** [comment]
![screenshot - 08102015 - 07 05 52 pm](https://cloud.githubusercontent.com/assets/11377070/9170149/f7137260-3f92-11e5-9aab-3db8b1c40be3.png)
em. It work! I use already patched fonts...Thanks

**Comment by dritter** [comment]
Good to hear! You are welcome :)

**Comment by dritter** [comment]
Btw. Could you mark this issue as resolved and close it? Thx.

**Comment by paulochf** [comment]
Hi! Can someone help me?

I'm having almost the same problem as @netcan's.

![term](https://cloud.githubusercontent.com/assets/107470/9364948/39cac470-4685-11e5-95ae-17dc4e74e20c.png)

Then I saw @dritter's comment

> In awesome-patched mode you need either a font that combines the original awesome font, octoicons and a powerline-patched font or a mechanism to use a normal font and fall back to the right one. The awesome-terminal-fonts repository suggests the latter way, but offers in its patching-strategy a already combined font (first way).
> If the second way doesn't work for you, just download one of the already patched fonts here.
> 
> I assume, you just installed the "awesome-font" from the "fonts" directory, which is just the normal awesome-font and does not include octoicons and powerline..

But I didn't figure out how to resolve this issue.

I'm using zsh on gnome-terminal, and the cited fonts from gabrielelana/awesome-terminal-fonts@c24e7e1e8c2106ef8a136b0861c11ce95b555443 (**patching-strategy** branch) were copied to .fonts but don't appear on font selection.

Thanks!

**Comment by dritter** [comment]
Hi @paulochf !

It seems like you mixed two things together. 

Either you install `awesome-terminal-fonts` with `fontcache` (**normal installation**). Then you have to copy the fallback-fonts to `~/.fonts`.

Or you just install an already patched font (**patching-strategy**). Then all you have to do, is to download one of the three fonts from [here](https://github.com/gabrielelana/awesome-terminal-fonts/tree/patching-strategy/patched) and install it (usually something like doubleclicking it, and clicking at "install font").

**Comment by paulochf** [comment]
Hey @dritter!

Ok, so (I guess) I removed the `awesome-terminal-fonts` and tried the **patching-strategy**. Downloaded the 3 fonts from your link to `~/.fonts`. When double-clicked, it opened the Font Viewer and there's a grey shaded button "Installed".

So, I went to `~/.zshrc` and enabled the theme

```shell
ZSH_THEME="powerlevel9k/powerlevel9k"
POWERLEVEL9K_MODE="awesome-patched"
```

but then

![zsh2](https://cloud.githubusercontent.com/assets/107470/9414414/b4e723c6-480e-11e5-999a-ed6db415ec7b.png)

Did I miss anything?
